packages/babel-preset-es2015/test/fixtures/traceur/ArrowFunctions/SloppyArguments.js

```diff
@@ -1,15 +1,15 @@
 
-var f1 = implements => implements;
-var f2 = implements => { return implements; };
-var f3 = (implements) => { return implements; };
+var f1 = i => i;
```

Comment:
These changes seem incorrect. Isn't the whole point of this test file to check sloppy arguments in arrow functions?